rules/xcodeproj.bzl

```diff
@@ -44,6 +45,20 @@ def _srcs_info_build_files(ctx):
 
     return [path]
 
+def _xcodeproj_aspect_collect_hmap_paths(deps, target, ctx):
+    hmap_paths = []
+    kind = ctx.rule.kind
+    is_library = kind == "objc_library" or kind == "swift_library"
+    if is_library:
+        for dep in deps:
+            if type(dep) == "Target" and HeaderMapInfo in dep:
```

Comment:
this `type` check shouldn't be necessary

Reply:
got `unsupported binary operation: Provider in NoneType`, should i do sth like `if type(dep) != "NoneType"`? Tried this and still have same error 😢

Reply:
sounds like we're somehow passing in a list that includes `None` into the list of `deps`, and we probably should filter that out ahead of time?